src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,7 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 * Remove TrustedAccess commands from aks-preview extension as it is GA and exists in azure-cli for long time.
+* Remove "--enable-pod-security-policy" as it's deprecated.
```

Comment:
Please mark this as a breaking change.
1. Add `[BREAKING CHANGE]` at the beginning of your change note.
2. Choose a new major version number like `15.0.0b1`, and update the version in `setup.py`

Reply:
fixed